Automattic/woocommerce-payments
# Using Event Type and string literals for events

Fixes - see discussion in https://github.com/Automattic/woocommerce-payments/issues/8075

#### Changes proposed in this Pull Request









* Using string literals for event names for better searchability/visibility of events.
* Defining an Event type for IDE auto-completion of event names and to reduce risk of typo errors.

#### Testing instructions





* No functionality should be affected by this change. 
* To test, follow the instructions from #8106 to ensure the events are firing as expected.

-------------------


**Post merge**
